BedrockServer.cpp

```diff
@@ -2356,3 +2356,9 @@ void BedrockServer::waitForHTTPS(unique_ptr<BedrockCommand>&& command) {
 const atomic<SQLiteNodeState>& BedrockServer::getState() const {
     return _nodeStateSnapshot == SQLiteNodeState::UNKNOWN ? _replicationState : _nodeStateSnapshot;
 }
+
+void BedrockServer::notifyStateChangeToPlugins(SQLite& db, SQLiteNodeState newState) {
+    for (auto plugin : plugins) {
+        plugin.second->stateChanged(db, newState);
+    }
+}
```

Comment:
EOL

Reply:
Why does that even matter?

Reply:
This is a standardization we apply across our entire codebase, and github shows this
<img width="725" alt="image" src="https://github.com/Expensify/Bedrock/assets/5258036/b47fa4b2-9147-46ab-a894-d3187a2c6446">

Reply:
*EOF

Reply:
> This is a standardization we apply across our entire codebase

Why? And why is it not handled by the linter?

> and github shows this

Who cares?